coder_sniffer/Drupal/Sniffs/NamingConventions/ValidAttributeHookNameSniff.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * \Drupal\Sniffs\NamingConventions\ValidAttributeHookNameSniff.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+namespace Drupal\Sniffs\NamingConventions;
```

Comment:
In the naming conventions folder we have PHP names, so I don't think the property string of an attribute belongs there.

Let's make a new folder "Attributes" and just call the class "ValidHookNameSniff"